testing/lightdm/APKBUILD

```diff
@@ -16,69 +16,49 @@ install="$pkgname.pre-install"
 pkgusers="lightdm"
 pkggroups="lightdm"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
```

Comment:
Please add `$pkgname-openrc` to the subpackages.

Reply:
Done.